js/api/Surveys.js

```diff
@@ -102,6 +106,31 @@ export function loadSurveyList(done) {
   });
 }
 
+/**
+ * Refreshes all of the data of accepted surveys, including Questions and Triggers
+ * @return {[type]} [description]
+ */
+export function refreshAcceptedSurveyData(surveyId) {
+  loadAllAcceptedSurveys((err, results) => {
+    if (surveyId) {
+      const surveys = _.filter(results, (survey) => { return survey.id === surveyId });
+      const survey = surveys[0];
+
+      if (survey) {
+        // loadParseFormDataBySurveyId(survey.id);
+      }
+    } else {
+      const resultLength = results.length;
+      try {
+        for (var i = 0; i < resultLength; i++) {
+          loadParseFormDataBySurveyId(results[i].id)
```

Comment:
`loadParseFormDataBySurveyId` doesn't seem to throw any errors?  What are the situations where this fails?